lib/CacheListener.php

```diff
@@ -9,8 +9,8 @@
 
 use OCA\GroupFolders\Mount\GroupFolderStorage;
 use OCP\EventDispatcher\IEventDispatcher;
-use OCP\Files\Cache\CacheInsertEvent;
-use OCP\Files\Cache\CacheUpdateEvent;
+use OCP\Files\Cache\CacheEntryInsertedEvent;
+use OCP\Files\Cache\CacheEntryUpdatedEvent;
```

Comment:
Do you have a ref for this change in server? Would make for a nice rector.

Reply:
https://github.com/nextcloud/server/pull/24113